src/main/java/bio/terra/command/CommandUtils.java

```diff
@@ -186,4 +186,36 @@ public static void printPolicyResponse(PolicyResponse policyResponse) {
         }
     }
 
+    /**
+     * Checks for valid format values (text and json are supported currently).
+     * Sets the default value to text if the format is null.
+     * Prints an error message to stdout and terminates the process if format is invalid.
+     * @param format typically the argument passed in on the command line
+     * @return the validated format value, which will only be changed if the default value is used
+     */
+    public static String validateFormat(String format) {
+        if (format == null) {
+            // set the default value to text
+            format = "text";
+        } else {
+            if (!StringUtils.equalsIgnoreCase(format, "text") && !StringUtils.equalsIgnoreCase(format, "json")) {
```

Comment:
would it be overkill to have enums for formats? (probably, right?)

Reply:
I think that's a good idea because the flags are currently used in two places. I added an enum to the CommandUtils class so that those two places and any future uses don't get out of sync with the flag values. Thanks for the suggestion!